packages/patternfly-react/src/components/SessionTimeout/helpers.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Session extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      now: this.props.now(),
+      expiresAt: this.props.expiresAt
+    };
+  }
+
+  componentWillMount() {
```

Comment:
componentWillMount  is being deprecated. It should also be avoided for async things. It is not guarantee that the component mounted. componentDidMount should be used.